indexers/keyvalue/LevelDBIndexer/tests/test_leveldbindexer.py

```diff
@@ -64,8 +64,9 @@ def apply_actions(save_abspath, index_abspath, actions):
 
 def validate_positive_results(keys, documents, searcher):
     assert len(list(zip(keys, documents))) > 0
+
```

Comment:
I'd rather have `result_docs = search.query(keys)` and iterate with zip with `query_docs` and `result_docs`

Reply:
Done